lib/tpm2_nv_util.h

```diff
@@ -1,5 +1,5 @@
 //**********************************************************************;
-// Copyright (c) 2017, Intel Corporation
+// Copyright (c) 2015-2018, Intel Corporation
```

Comment:
2017-2018

Reply:
I was told that changes to copyright need to be in a separate commit.  Is this true?

Reply:
@williamcroberts @danintel will make a separate commit.